en/docs/next/ai-workspace/policies/build-gateway-with-ai-policies.md

```
@@ -0,0 +1,193 @@
+| Field | Description |
+|---|---|
+| `gomodule` | Go module reference for policy hub managed policies (e.g., `github.com/wso2/gateway-controllers/policies/pii-masking@v1`) |
+| `filePath` | Relative path from `build.yaml` to a local custom AI policy directory |
+
+### Adding a Custom AI Policy
+
+Your custom AI policy can reside anywhere on the filesystem. Use a relative path from the `build.yaml` file to point to it.
+
+For example, if your directory layout is:
+
+```text
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)